### PR TITLE
Fixed the resolver syntax in examples.

### PR DESCRIPTION
**Pull request by ni3gavhane**
### Summary
The example folder had wrong syntax for Resolver.
Comment in DIDDocumentLite and did-ethr-resolevr also had wrong Resolver syntax in the comment.
#### Type of request
Fix the syntax.

### CODE REVIEW

File: examples/private claim workflow.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ stored and verified
   }); 
   const issuerAddress = '0xddCe879DE01391176a8527681f63A7D3FCA2901B'; 
   const issuerDid = `did:${Methods.Ethereum}:${issuerAddress}` ;
```
Comment (drgorb) [inline examples/private claim workflow.md:54]: `Methods` enum does not have an `Ethereum` member anymore

**Comment by drgorb** [comment]
are the examples still correct with all the changes in the last 2 weeks?

**Comment by ni3gavhane** [comment]
> are the examples still correct with all the changes in the last 2 weeks?

**Comment by ni3gavhane** [comment]
yes, the examples are right.

**Pull request merged**